docs/specs/ANALYTICS.md

```
@@ -805,6 +805,261 @@ QoT = SUM(teammate_rating * shared_toi) / SUM(shared_toi)
+| Scoring | goals_per_60 (3.0), xg_per_60 (2.0), finishing_skill (1.5), shooting_pct (1.0), hd_for_per_60 (1.5) | Goal-scoring ability |
+| Playmaking | assists_per_60 (2.5), shot_assists_per_60 (2.0), pass_completion_pct (1.5), zone_ent_controlled_per_60 (1.5), creative_passes_per_60 (1.0) | Passing and creation |
+| Shot Generation | shots_per_60 (2.0), sog_per_60 (2.0), sc_for_per_60 (1.5), rush_shots_per_60 (1.0) | Volume shooting and chances |
+| Transition | zone_ent_control_pct (2.0), zone_exit_success_pct (2.0), zone_ent_per_60 (1.5), zone_ext_per_60 (1.5), rush_involvement_per_60 (1.0) | Puck movement between zones |
+| Defense | takeaways_per_60 (+2.0), blocks_per_60 (+1.5), poke_checks_per_60 (+1.0), giveaways_per_60 (-2.0), ca_per_60 (-1.5) | Defensive play (mixed-sign) |
+| Compete | forechecks_per_60 (2.0), backchecks_per_60 (2.5), puck_battles_per_60 (1.5), hits_per_60 (1.0), loose_puck_wins_per_60 (1.5), board_battles_won_per_60 (1.0) | Effort and physicality |
+
+**Clutch Score** (separate from 6 dimensions):
+trailing_goals_per_60 (2.5), p3_goals_per_60 (2.0), close_game_cf_pct (1.5), p3_clutch_diff (1.5), trailing_cf_pct (1.0)
+
+**3 Composite Rollups:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)